pusher/src/Services/SocketManager.ts

```diff
@@ -227,28 +216,15 @@ export class SocketManager implements ZoneEventListener {
                     }
                 })
                 .on("end", () => {
-                    console.warn(
-                        "Connection lost to back server '" +
-                            apiClient.getChannel().getTarget() +
-                            "' for room '" +
-                            client.roomId +
-                            "'"
-                    );
+                    console.warn("Connection lost to back server");
```

Comment:
You are removing a commit I made to add more details here. Are you sure you did not force push something not intended?